integration_test/tests/helper/Expect.ts

```diff
@@ -0,0 +1,208 @@
+      else {
+        const errorMessage = `Expected ${actual} to be null`;
+        console.error(`[EXPECTATION NOT MET]: ${errorMessage}`);
+        throwErr(errorMessage);
+      }
+    },
+
+    toBeNotNull() {
+      if (actual !== null) logPass(`${actual} is not null`);
+      else {
+        const errorMessage = `Expected ${actual} to be not null`;
+        console.error(`[EXPECTATION NOT MET]: ${errorMessage}`);
+        throwErr(errorMessage);
+      }
+    },
+
+    toBeInstanceOf(expected: any) {
+      if (actual instanceof expected)
+        logPass(`${actual} is instance of ${expected.name}`);
+      else {
+        const errorMessage = `Expected ${actual} to be instance of ${expected.name}`;
+        console.error(`[EXPECTATION NOT MET]: ${errorMessage}`);
+        throwErr(errorMessage);
+      }
+    },
+
+    toContain(expected: any) {
+      if (actual && actual.includes && actual.includes(expected))
+        logPass(`${actual} contains ${expected}`);
+      else {
+        const errorMessage = `Expected ${actual} to contain ${expected}`;
+        console.error(`[EXPECTATION NOT MET]: ${errorMessage}`);
+        throwErr(errorMessage);
+      }
+    },
+
+    toHaveLength(expected: number) {
+      if (actual && actual.length === expected)
+        logPass(`${actual} has length ${expected}`);
+      else {
+        const errorMessage = `Expected ${actual} to have length ${expected}, but got ${
+          actual ? actual.length : 'undefined'
+        }`;
+        console.error(`[EXPECTATION NOT MET]: ${errorMessage}`);
+        throwErr(errorMessage);
+      }
+    },
+
+    toHaveProperty(property: string) {
+      if (actual && actual.hasOwnProperty(property))
+        logPass(`${actual} has property '${property}'`);
+      else {
+        const errorMessage = `Expected ${actual} to have property '${property}'`;
+        console.error(`[EXPECTATION NOT MET]: ${errorMessage}`);
+        throwErr(errorMessage);
+      }
+    },
+
+    toHavePropertyValue(property: string, value: any) {
+      if (
+        actual &&
+        actual.hasOwnProperty(property) &&
+        actual[property] === value
+      ) {
+        logPass(`${actual} has property '${property}' with value ${value}`);
+      } else {
+        const actualValue = actual ? actual[property] : 'undefined';
+        const errorMessage = `Expected ${actual} to have property '${property}' with value ${value}, but got ${actualValue}`;
+        console.error(`[EXPECTATION NOT MET]: ${errorMessage}`);
+        throwErr(errorMessage);
+      }
+    },
+
+    toMatchRegex(regex: RegExp) {
+      if (regex.test(actual)) logPass(`${actual} matches regex ${regex}`);
+      else {
+        const errorMessage = `Expected ${actual} to match regex ${regex}`;
+        console.error(`[EXPECTATION NOT MET]: ${errorMessage}`);
+        throwErr(errorMessage);
+      }
+    },
+
+    toBeCloseTo(expected: number, precision: number = 2) {
```

Comment:
Nit: here as well, we could check for `actual` to be a number before doing any math:

```suggestion
    toBeCloseTo(expected: number, precision: number = 2) {
        if (typeof actual !== 'number') {
            throwErr(`Both ${actual} and ${expected} must be numbers`);
        }
```